core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts

```diff
@@ -158,5 +209,37 @@ describe('DotContainerPropertiesComponent', () => {
             expect(inplace.componentInstance.active).toBe(true);
             expect(title.attributes.dotAutofocus).toBeDefined();
         });
+
+        it('should setup title', () => {
+            const field = de.query(By.css('[data-testId="title"]'));
+            expect(field.attributes.pInputText).toBeDefined();
+        });
+
+        it('should setup description', () => {
+            const field = de.query(By.css('[data-testId="description"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should setup Max Contents', () => {
+            const field = de.query(By.css('[data-testId="max-contents"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should setup code', () => {
+            const field = de.query(By.css('[data-testId="code"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should button enable when max content greater then zero', fakeAsync(() => {
+            fixture.componentInstance.form.get('maxContentlets').setValue(2);
+            const contentTypeButton = de.query(By.css('[data-testId=showContentTypeAndCode]'));
+            contentTypeButton.nativeElement.click();
```

Comment:
https://angular.io/guide/testing-components-scenarios#clicking

Reply:
Check this doc please, Angular have a specific way to testing events.

Reply:
I already updated it accordingly. would you please review it?

Reply:
Would you please check again it doesn't return `EventEmitter` so we don't need to subscribe event it should simply click and called the function which shows the container content types.